control_plane/attachment_service/src/main.rs

```diff
@@ -55,11 +54,31 @@ struct Cli {
     #[arg(long)]
     database_url: Option<String>,
 
+    /// Flag to enable dev mode, which permits running without auth
+    #[arg(long, default_value = "false")]
+    dev: bool,
+
     /// Grace period before marking unresponsive pageserver offline
     #[arg(long)]
     max_unavailable_interval: Option<humantime::Duration>,
 }
 
+enum StrictMode {
```

Comment:
nit


```suggestion
    database_url: Option<String>,

    /// Flag to enable dev mode, which permits running without auth
    #[arg(long, default_value = "strict")]
    strictness: StrictMode,

    /// Grace period before marking unresponsive pageserver offline
    #[arg(long)]
    max_unavailable_interval: Option<humantime::Duration>,
}

#[derive(clap::ValueEnum)]
enum StrictMode {
```

Reply:
While we internally use enums to avoid bare booleans, CLI flags that have only two possible states are more conventionally expressed as a flag `--dev` rather than having to `--strictness=dev`.  The difference doesn't matter much here, as it's just for tests, but I prefer to keep the CLI part as a bool.